src/components/AboutUs/AboutUs.jsx

```diff
@@ -22,7 +23,22 @@ const bios = [
   'Web Developer',
   'Web Developer',
 ]
-
+const github = [
+  'https://github.com/maqalaqil',
+  'https://github.com/murtaja1',
+  'https://github.com/ZhinAbubakr',
+  'https://github.com/Othmanosx',
+  'https://github.com/Ravyar19',
+  'https://github.com/JalalArif',
+]
+const linkedin = [
+  'https://www.linkedin.com/in/maher-al-aqil-56895894/',
+  'https://www.linkedin.com/in/murtaja-adnan-2a02b9206/',
+  'https://www.linkedin.com/in/zhin-abubakr-473910181/',
+  'https://www.linkedin.com/in/othmanosx/',
+  'https://www.linkedin.com/in/ravyar-aram-b767661aa/',
+  'https://www.linkedin.com/in/jalarif/',
+]
```

Comment:
names, teamImg, github, linkedin
those arrays could have been a single array of objects like this:
persons = [
  {
    name: 'name',
    img: img,
    github: 'github.com',
    linkedin: 'linkedin.com',
  },
  {
    name: 'name',
    img: img,
    github: 'github.com',
    linkedin: 'linkedin.com',
  },
  {
    name: 'name',
    img: img,
    github: 'github.com',
    linkedin: 'linkedin.com',
  },
  {
    name: 'name',
    img: img,
    github: 'github.com',
    linkedin: 'linkedin.com',
  },
]

Reply:
you are very right and I know this not the best way to do as some other things, but sometimes I do it very quickly to save the time.
but I will change it.